anchors.go

```diff
@@ -0,0 +1,132 @@
+package factom
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// Anchors is an anchors response from factomd.
+// Note that Ethereum or Bitcoin can be nil
+type Anchors struct {
+	Height   uint32          `json:"directoryblockheight"`
+	KeyMR    string          `json:"directoryblockkeymr"`
+	Bitcoin  *AnchorBitcoin  `json:"bitcoin"`
+	Ethereum *AnchorEthereum `json:"ethereum"`
+}
+
+// AnchorBitcoin is the bitcoin specific anchor
+type AnchorBitcoin struct {
+	TransactionHash string `json:"transactionhash"`
+	BlockHash       string `json:"blockhash"`
+}
+
+// AnchorEthereum is the ethereum specific anchor
+type AnchorEthereum struct {
+	RecordHeight int64        `json:"recordheight"`
+	DBHeightMax  int64        `json:"dbheightmax"`
+	DBHeightMin  int64        `json:"dbheightmin"`
+	WindowMR     string       `json:"windowmr"`
+	MerkleBranch []MerkleNode `json:"merklebranch"`
+
+	ContractAddress string `json:"contractaddress"`
+	TxID            string `json:"txid"`
+	BlockHash       string `json:"blockhash"`
+	TxIndex         int64  `json:"txindex"`
+}
+
+// MerkleNode is part of the ethereum anchor
+type MerkleNode struct {
+	Left  string `json:"left,omitempty"`
+	Right string `json:"right,omitempty"`
+	Top   string `json:"top,omitempty"`
+}
+
+func (a *Anchors) String() string {
```

Comment:
Do we need to build by hand this string? Isn't built in [JSON pretty print](https://golang.org/pkg/encoding/json/#MarshalIndent) good enough? Would avoid a lot of boiler plate code and would be more future proof

Reply:
that's a good question, i was just copying what was done in the other functions. i'll check out pretty print to see if we can consolidate all that in a new pr

Reply:
Created an issue for tracking: https://github.com/FactomProject/factom/issues/143